File: Swift/프로그래머스/1/12901. 2016년/2016년.swift
func solution(_ a:Int, _ b:Int) -> String {
    let week = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"]
    let month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    
    let weekIndex = month[0..<a-1].reduce(0, +) + b
    
    return week[(weekIndex - 1) % 7]
}

File: Python3/프로그래머스/1/12901. 2016년/2016년.py
def solution(a, b):
    week = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"]
    month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    
    num = sum(month[0:a-1]) + b

    
    return week[num % len(week) - 1]